docs/docsite/rst/user_guide/become.rst

```diff
@@ -72,7 +72,7 @@ Become connection variables
 You can define different ``become`` options for each managed node or group. You can define these variables in inventory or use them as normal variables.
 
 ansible_become
-    equivalent of the become directive, decides if privilege escalation is used or not.
+    equivalent of the ``become`` directive, decides if privilege escalation is used or not.
```

Comment:
not an issue with the change, but the original phrasing was already 'wrongish' .. its not equivalent, it overrides